core-data/resolvers.js

```diff
@@ -28,3 +33,14 @@ export async function* getMedia( state, id ) {
 	const media = await apiRequest( { path: `/wp/v2/media/${ id }` } );
 	yield receiveMedia( media );
 }
+
+/**
+ * Requests a post type element from the REST API.
+ *
+ * @param {Object} state State tree
+ * @param {number} slug  Post Type slug
+ */
+export async function* getPostType( state, slug ) {
+	const media = await apiRequest( { path: `/wp/v2/types/${ slug }?context=edit` } );
```

Comment:
The context is hard-coded for now. As it's the only context we use right now.
I wonder how should we approach this:

 - Always use `edit` assuming this context returns the full object?
 - Store received data per context and make the context and argument of the resolver (I don't like this)
 - Something else (maybe server-side)?

What's your take on this?